### PR TITLE
Handle ProcessProductPackaging message

### PR DESCRIPTION
**Pull request by keith**
This message is printed when reading xcent files. It has an arbitrary
length depending on your entitlements. This currently has one issue
where it prints a newline after itself, because it exits on a specific
string after matching as many lines as possible between the start, and
there's no way to capture output after exiting.

### CODE REVIEW

File: lib/xcpretty/parser/chunks.rb [hunk 1]
```diff
@@ -140,6 +140,16 @@ module XCPretty
   c.line SHELL_BUILTIN
 end
 
+chunk "Process entitlements" do |c|
+  c.line /^ProcessProductPackaging "" (#{PATH})(?:-Simulated)\.xcent$/ do |f, m|
```
Comment (supermarin) [inline lib/xcpretty/parser/chunks.rb]: I think you can `^ProcessProductPackaging .* (#{PATH}... $` in case there's something in the quotes

Comment (keith) [reply lib/xcpretty/parser/chunks.rb]: yea, i was looking through our logs to try find something else and couldn't, will do that anyways.

File: lib/xcpretty/parser/chunks.rb [hunk 2]
```diff
@@ -140,6 +140,16 @@ module XCPretty
   c.line SHELL_BUILTIN
 end
 
+chunk "Process entitlements" do |c|
+  c.line /^ProcessProductPackaging .* (#{PATH})(?:-Simulated)\.xcent$/ do |f, m|
+    f.format_process_entitlements(path(m[1]).basename.to_s)
+  end
+  c.exit /^\s{4}builtin-productPackagingUtility.*/
```
Comment (supermarin) [inline lib/xcpretty/parser/chunks.rb]: $ on the end or \n

Comment (keith) [reply lib/xcpretty/parser/chunks.rb]: done

**Comment by supermarin** [comment]
awesome PR, tnx

**Comment by dduan** [comment]
awesome

File: lib/xcpretty/parser/chunks.rb [hunk 3]
```diff
@@ -140,6 +140,16 @@ module XCPretty
   c.line SHELL_BUILTIN
 end
 
+chunk "Process entitlements" do |c|
+  c.line /^ProcessProductPackaging .* (#{PATH})(?:-Simulated)\.xcent$/ do |f, m|
```
[the remark this answers is not in the document]
Comment (kastiglione) [reply lib/xcpretty/parser/chunks.rb]: Was `(?:-Simulated)` intended to be `?`

[the remark this answers is not in the document]
Comment (keith) [reply lib/xcpretty/parser/chunks.rb]: You're right, fixing here https://github.com/supermarin/xcpretty/pull/333

**Pull request merged**